source/euclid.js

```diff
@@ -0,0 +1,15 @@
+'use strict';
+
+const gcd = (x, y) => y == 0 ? x : gcd(y, 	x % y)
```

Comment:
всё верно, но проверка на 0 не нужна - он сам будет `false` - а всё остальное `true` + форматирование - убрать затесавшийся таб
```suggestion
const gcd = (x, y) => y ? gcd(y, x % y) : x;
```